content/applications/inventory_and_mrp/barcode/setup/device_troubleshooting.rst

```diff
@@ -0,0 +1,207 @@
+
+Barcode app doesn't give feedback
+---------------------------------
+
+By default, Android barcode scanners pre-process the barcode, then send a full text. Since Odoo
+*Barcode* doesn't read this type of output, settings for each type of scanner **must** be configured
+correctly.
+
+Odoo *Barcode* expects that the scanner works like an analog keyboard, and so, only detects *key
+events*. Refer to the following sections for configuration settings for the most popular devices.
+
+Zebra TC21/TC26
+---------------
+
+When using Zebra scanners, ensure the following keystroke configurations are set to prevent errors.
+
+Begin on the Zebra scanner's home screen, and select the :guilabel:`DataWedge` app (the app is
+represented by a :guilabel:`light blue barcode` icon).
+
+On the :guilabel:`DataWedge Profiles` page, select the profile option to access the Zebra scanner's
+settings.
+
+Once the profile is selected, scroll down to the :guilabel:`Keyboard Output` option, and ensure the
+:guilabel:`Enable/disable keystroke output` option is :guilabel:`Enabled`.
+
+.. image:: device_troubleshooting/device-troubleshooting-zebra-settings.png
+   :align: center
+   :alt: Show keystroke option in the Zebra scanner's DataWedge app.
+
+Once that option is enabled, go back to the :guilabel:`Profile` options page, and go to the
+:guilabel:`Keystroke output` section. Then, open the :guilabel:`Key event options` submenu. Under
+:guilabel:`Characters`, ensure the :guilabel:`Send Characters as Events` option is checked.
+
+.. important::
+   The :guilabel:`Send Characters as Events` option **must** be checked on the Zebra scanner, or
+   Odoo **cannot** recognize the barcodes that are scanned.
+
+Once the above steps have been taken, perform a test scan to ensure the Zebra scanner is working as
+intended.
+
+MUNBYN Android devices
+----------------------
+
+When using MUNBYN Android scanners, ensure the following configurations are set to prevent errors.
+
+From the device's home screen, click :menuselection:`AppSettings`. On the resulting page, locate the
+:guilabel:`Process mode` section, and select :guilabel:`Keyboard input`.
+
+.. image:: device_troubleshooting/device-troubleshooting-munbyn-process-mode.png
+   :align: center
+   :alt: Process mode section on MUNBYN scanner's AppSettings page.
+
+.. tip::
+   The selected *Process mode* controls how data will be processed after barcode data has been read
+   out. *Keyboard input* enters read-out data at the position of the cursor, the same as input data
+   on an analog keyboard would.
+
+Once the above steps have been taken, perform a test scan to ensure the MUNBYN Android scanner is
+working as intended.
+
+.. admonition:: Why is there no data output in the app after a successful scan?
+
+   When scanning a barcode, the scanner might beep, indicating a successful scan, but there is no
+   data output in the app.
+
+   To fix this issue, adjust the output method to *keyboard analogue* in the *Scanner* app on the
```

Comment:
please ensure that the correct spelling of "analogue" is used here (analogue or analog)

Reply:
Both are correct spellings, analog being preferred in the US and analogue in the UK. In this instance, *analogue* is correct. However, to avoid confusion, I'll just change it to "analogue" throughout the doc entirely

Reply:
Suggestion implemented ✅